modules/manage/pages/schema-reg/schema-reg-authorization.adoc

```diff
@@ -0,0 +1,385 @@
+
+* `REGISTRY`: Controls whether or not to grant ACL access to global, or top-level Schema Registry operations. Specify using the flag `registry-global`.
+* `SUBJECT`: Controls ACL access for specific Schema Registry subjects. Specify using the flag `registry-subject`.
+
+== Supported operations
+
+Redpanda Schema Registry ACLs support the following specific subset of Schema Registry endpoints and operations:
+
+[NOTE]
+====
+Not all Kafka operations are supported when using Redpanda Schema Registry ACLs.
+====
+
+|===
+| Endpoint | HTTP Verb | Operation | Resource
+
+| `/config`
+| `GET`
+| `describe_configs`
+| `registry-global`
+
+| `/config`
+| `PUT`
+| `alter_configs`
+| `registry-global`
+
+| `/config/{subject}`
+| `GET`
+| `describe_configs`
+| `registry-subject`
+
+| `/config/{subject}`
+| `PUT`
+| `alter_configs`
+| `registry-subject`
+
+| `/config/{subject}`
+| `DELETE`
+| `alter_configs`
+| `registry-subject`
+
+| `/mode`
+| `GET`
+| `describe_configs`
+| `registry-global`
+
+| `/mode`
+| `PUT`
+| `alter_configs`
+| `registry-global`
+
+| `/mode/{subject}`
+| `GET`
+| `describe_configs`
+| `registry-subject`
+
+| `/mode/{subject}`
+| `PUT`
+| `alter_configs`
+| `registry-subject`
+
+| `/mode/{subject}`
+| `DELETE`
+| `alter_configs`
+| `registry-subject`
+
+| `/schemas/types`
+| `GET`
+| none/open
+| -
+
+| `/schema/ids/{id}`
+| `GET`
+| `read`
+| `registry-subject`
+
+| `/schema/ids/{versions}`
+| `GET`
+| `describe`
+| `registry-global`
+
+| `/schema/ids/{subjects}`
+| `GET`
+| `describe`
+| `registry-subject`
+
+| `/subjects`
+| `GET`
+| `describe`
+| `registry-subject`
+
+| `/subjects/{subject}`
+| `POST`
+| `read`
+| `registry-subject`
+
+| `/subjects/{subject}`
+| `DELETE`
+| `delete`
+| `registry-subject`
+
+| `/subjects/{subject}/versions`
+| `GET`
+| `describe`
+| `registry-subject`
+
+| `/subjects/{subject}/versions`
+| `POST`
+| `write`
+| `registry-subject`
+
+| `/subjects/{subject}/versions/{version}`
+| `GET`
+| `read`
+| `registry-subject`
+
+| `/subjects/{subject}/versions/{version}`
+| `DELETE`
+| `delete`
+| `registry-subject`
+
+| `/subjects/{subject}/versions/schema`
+| `GET`
+| `read`
+| `registry-subject`
+
+| `/subjects/{subject}/versions/referencedby`
+| `GET`
+| `describe`
+| `registry-global`
+
+| `compatibility/subjects/{subject}/versions/{version}`
+| `POST`
+| `read`
+| `registry-subject`
+
+| `/status/ready`
+| `GET`
+| none/open
+| -
+
+| `/security/acls`
+| `GET`
+| `describe`
+| `cluster_action`
+
+| `/security/acls`
+| `POST`
+| `alter`
+| `cluster_action`
+
+| `/security/acls`
+| `DELETE`
+| `alter`
+| `cluster_action`
+|===
+
+For additional guidance on these operations, see the xref:api:ROOT:schema-registry-api.adoc#get-/security/acls[Redpanda Schema Registry API].
+
+== Enable Schema Registry Authorization
+
+=== Prerequisites
+
+Before you can enable Schema Registry Authorization, you must have:
+
+* A valid Redpanda Enterprise license
+* v25.2 version of rpk
```

Comment:
```suggestion
* `rpk` version 25.2+
```
You could also add: To check your current version, see <link to Install or Update rpk>